### PR TITLE
Open Source

### PR DESCRIPTION
**Pull request by BigBlueHat**
These changes effectively open source this project under the
[BSD-3-Clause](https://opensource.org/license/bsd-3-clause/) license.

Additionally it follows most of the REUSE v3.0 specification which outlines
how license claims can be made unambiguously for every file.

Happy to discuss the particulars!
:tophat:

### CODE REVIEW

File: lib/events.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /*!
- * Copyright (c) 2012-2022 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2012-2024 Digital Bazaar, Inc.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```
Comment (dlongley) [inline lib/events.js:4]: Bummer that we need this in every file.

**Review by dlongley — APPROVED** [review]
Thanks! @davidlehn, this look right to you?

**Comment by dlongley** [comment]
Oh, we need a changelog entry as well, with a minor release bump, I think. We could also bump the node version to 18+ while we're at it.

File: lib/events.js (hunk 1 above)
Comment (BigBlueHat) [reply lib/events.js:4]: There is an alternative for spelling out copyright in a `.reuse/dep5` file--which I've added for formats where comments are cumbersome/odd (Markdown uses HTML comments) or not available (`.json`): https://github.com/digitalbazaar/bedrock/pull/112/commits/45a2af2649ea2c46d73b309763aea52486a343c2

The more explicit we are, the less confusion the could/would be when we include any outside contribution, file, or directory (which would contain it's own copyright info).

The wildcard approach of a single `LICENSE` file just isn't as clear.

**Comment by BigBlueHat** [comment]
> Oh, we need a changelog entry as well, with a minor release bump, I think. We could also bump the node version to 18+ while we're at it.

Do you want those Node version bumps (in package.json and the workflow) done in a separate PR?

**Comment by BigBlueHat** [comment]
This PR properly licenses every file in this repo (per the [RESUE standard](https://reuse.software/) as examined by the [reuse tool](https://git.fsfe.org/reuse/tool):

```
# SUMMARY

* Bad licenses: 0
* Deprecated licenses: 0
* Licenses without file extension: 0
* Missing licenses: 0
* Unused licenses: 0
* Used licenses: BSD-3-Clause
* Read errors: 0
* files with copyright information: 29 / 29
* files with license information: 29 / 29

Congratulations! Your project is compliant with version 3.0 of the REUSE Specification :-)
```

**Comment by dlongley** [comment]
> > Oh, we need a changelog entry as well, with a minor release bump, I think. We could also bump the node version to 18+ while we're at it.
> 
> Do you want those Node version bumps (in `package.json` and the workflow) done in a separate PR?
> 
> The workflow actions also need version bumps...

They can be done here, that's fine -- another PR is also fine, your call.

**Comment by dlongley** [comment]
We still need a CHANGELOG entry here with the license update -- and the README mentions the license in it ... that might need an update.

**Comment by BigBlueHat** [comment]
> We still need a CHANGELOG entry here with the license update -- and the README mentions the license in it ... that might need an update.

My bad! I'd thought I'd committed and pushed that...but wandered off to do the node stuff. Sorry about that.

File: LICENSE-bedrock.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+Bedrock Non-Commercial License v1.0
+===================================
+
+Copyright (c) 2011-2021 Digital Bazaar, Inc.
```
Comment (dlongley) [inline LICENSE-bedrock.md:4]:
```suggestion
Copyright (c) 2011-2024 Digital Bazaar, Inc.
```

File: package.json [hunk 3]
```diff
@@ -43,7 +43,7 @@
     "jsdoc-to-markdown": "^7.1.1"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=20"
```
Comment (dlongley) [inline package.json:46]: 18 is still being maintained so we can include that here.

```suggestion
    "node": ">=18"
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 ## 6.0.2 - xxxx-xx-xx
 - Open Source under BSD-3-Clause license.
+- Bump Node.js versions to 20.x (current LTS).
```
Comment (dlongley) [inline CHANGELOG.md:5]:
```suggestion
- Bump Node.js versions to 18.x (LTS).
```

**Review by dlongley — APPROVED** [review]
See couple of suggestions and then we're good.

**Pull request merged**